preprocessing.py

```diff
@@ -227,11 +228,12 @@ def randomly_sample_aug_data(aug_data, k, augs=25):
     # load parsed midi files
     if not args.from_augmented_data:
         print("Translating midi files to event vocabulary (NOTE: may take a while)...") if args.verbose else None
-        for file in os.listdir(PATH):
+        for file in glob.iglob(PATH + '**/*.mid*', recursive=True):
+            print(file) if args.verbose else None
             try:
-                idx_list = midi_parser(fname=PATH + file)[0]
+                idx_list = midi_parser(fname=file)[0]
                 DATA.append(idx_list)
-            except OSError:
+            except (OSError, ValueError, EOFError, IndexError, TypeError, KeyError, RuntimeError, AttributeError):
                 pass
```

Comment:
How come so many errors are excepted? Is this the actual list of errors that glob may raise? If so, I'd prefer that you except the error as `e`, then print it out before passing

Reply:
Ok I admit this one was a bit excessive, the crash was with the dataset I was using. It crashed the midi parser. This just caught and ignored them. I will clean this up.

Reply:
Cleaned up to only exceptions when running the midi parser.